examples/sw/tmd.sw

```diff
@@ -0,0 +1,143 @@
+# DATE: 2018-03-26 UNITS: metal CONTRIBUTOR: Jin-Wu Jiang jwjiang5918@hotmail.com
```

Comment:
Because of the requested renaming of the pair style, this potential file should also have a different suffix

Reply:
renamed tmd.sw to tmd.sw.mod.

Reply:
renamed but not deleted. this is fixed now, too.